### PR TITLE
Console can show/set properties of selected pieces

### PR DESCRIPTION
**Pull request by Cattlesquat**
Request from @riverwanderer

By adding "selected" into the command line the properties of currently selected pieces can be shown/set

`/property selected show LocationName`

### CODE REVIEW

**Review by BrentEaston — COMMENTED** [review]
I see no security issues as console commands are only allowed in the 'only ever single user' case.

One thing, many properties, especially trait properties cannot be set, is there anything in the doco to note this since setting a non-existent or unsettable property just fails quietly?

**Comment by Cattlesquat** [comment]
The Console, so far, doesn't much detail anywhere in the html help -- it's only documented under the "/help" command in the console itself -- so I have added text to that.

File: vassal-app/src/main/java/VASSAL/build/module/Console.java [hunk 1]
```diff
@@ -159,31 +163,74 @@ else if (matches("?", option) || matches("help", option) || ("".equals(option)))
...
-      if (propValue != null) {
-        show("[" + property + "]: " + propValue.getPropertyValue());
+    else if (matches("show", option) || matches("set", option)) { //NON-NLS
+
+      final GameModule gm = GameModule.getGameModule();
+
+      if (useSelected) {
+        final List<GamePiece> selected = KeyBuffer.getBuffer().asList();
+
+        if ((selected != null) && !selected.isEmpty()) {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/Console.java:193]: You don't need to check if it's non-empty here---the loop just won't loop over an empty list.

Comment (uckelman) [reply vassal-app/src/main/java/VASSAL/build/module/Console.java:193]: Oh, there's an `else` down there. Never mind!

Comment (uckelman) [reply vassal-app/src/main/java/VASSAL/build/module/Console.java:193]: Wait, no, that `else` is further out in scope.

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/build/module/Console.java]: fixed

**Pull request merged**